src/ringct/rctTypes.h

```diff
@@ -217,8 +217,7 @@ namespace rct {
       rct::keyV L, R;
       rct::key a, b, t;
 
-      Bulletproof():
-        A({}), S({}), T1({}), T2({}), taux({}), mu({}), a({}), b({}), t({}) {}
+      Bulletproof() = default;
```

Comment:
This changes the behavior, and now the keys are not initialized to zero (except for `V`) if `Bulletproof` is default constructed.

Reply:
Why would they not be initialized to zero? `rct::key` is a class type with no user provided constructor, and the default constructor of `Bulletproof` will call the implicit default constructor of `rct::key`, which will zero out the `bytes[32]` array.

Reply:
@vtnerd can you also confirm this is resolved?

Reply:
The default constructor does default initialization, not value initialization.

Reply:
https://godbolt.org/z/rMhd75PEn notice that there is no zero-initialization in the ASM.

Reply:
You're right, I *was* mixing up default initialization and value initialization

Reply:
Latest commit fixes that